core/modules/admin_bar/css/admin_bar.css

```
@@ -210,11 +210,18 @@
   margin: calc(-1px - var(--admin-bar-height)) 0 0 15em; /* LTR */
 }
 #admin-bar .dropdown li li.expandable.outside-right ul {
+  z-index: 1;
   margin-left: -15em; /* LTR */
+  box-shadow:
```

Let's add a code comment to describe the intention here:

```suggestion
  /* Add a shadow to help distinguish when a sub-menu overlaps its parent. */
  box-shadow:
```